packages/x-components/src/x-modules/search/components/__tests__/sort-dropdown.spec.ts

```diff
@@ -23,7 +23,6 @@ function renderSortDropdown({
       </template>
    </SortDropdown>`,
   items = ['Relevance', 'Price low to high', 'Price high to low'],
```

Comment:
Given that we agreed to have the default value be `''`, should we replace `'Relevance'` here with `''`?